### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by pandamicro**

### CODE REVIEW

**Comment by WenhaiLin** [comment]
[ci rebuild]

File: extensions/CocoStudio/GUI/BaseClasses/UIWidget.cpp [hunk 1]
```diff
@@ -131,12 +131,12 @@ void Widget::visit()
 
 void Widget::addChild(CCNode *child)
 {
-    CCNode::addChild(child);
+    addChild(child, 0, kCCNodeTagInvalid);
```
Comment (lihex) [inline extensions/CocoStudio/GUI/BaseClasses/UIWidget.cpp]: // should be this code, the other addChild method is similar
addChild(child, child->getZOrder(), child->getTag());

Comment (pandamicro) [inline extensions/CocoStudio/GUI/BaseClasses/UIWidget.cpp]: Thanks, resolved by new commits

**Comment by WenhaiLin** [comment]
[ci rebuild]

**Comment by pandamicro** [comment]
[ci rebuild]

**Pull request merged**